xbmc/windowing/gbm/drm/DRMUtils.cpp

```diff
@@ -368,20 +368,23 @@ bool CDRMUtils::OpenDrm(bool needConnector)
 
     PrintDrmDeviceInfo(device);
 
-    const char* renderPath = drmGetRenderDeviceNameFromFd(m_fd);
+    m_renderDevicePath = std::unique_ptr<const char, StrdupDeleter>(
```

Comment:
As much as I like custom deleters this seems like an odd one. I think I would rather convert to a `std::string`.

This could be done with a helper function that wraps the c methods and still free the `char*`.